packages/oauth-adapters/src/oAuthToken.ts

```diff
@@ -20,9 +26,10 @@ export interface OAuthToken {
    * Used to get a new access token when it expires.
    */
   refreshToken?: string;
+  [key: string]: unknown;
```

Comment:
Please test this change by calling fetchToken call in any other oauth spec given there OauthToken does not have additional parameters defined.